ROADMAP.md

```diff
@@ -4,4 +4,138 @@ Roadmap
 _Note: Previous roadmaps can be found with the roadmap reviews for that period.
 [Link to Roadmap 2022](roadmap-reviews/2022/ROADMAP.md)_
 
-2023 roadmap coming soon.
\ No newline at end of file
+This document spans the Roadmap for the in-toto project for the 2023 calendar
+year. As before, the theme of this year's efforts are focused on extending
+in-toto through new and existing ITEs, and adding support for them in the
+implementations we maintain.
+
+## in-toto v1.0
```

Comment:
Good idea, but what does a v1 mean for other versions (e.g., in-toto/attestation)?

Reply:
Added a note. Does this read okay?

```
Note that this planned v1.0 release checkpoints in-toto at its pre-attestation
stage. As the attestation framework matures and reaches stability, it is likely
to have its own tagged releases, either as a standalone specification or as part
of the main specification's future releases.
```